test/general/openpgp.js

```diff
@@ -2428,6 +2428,41 @@ describe('OpenPGP.js public api tests', function() {
         });
       });
 
+      it('should decrypt with revoked subkey', async function() {
+        const pubKeyDE = (await openpgp.key.readArmored(pub_key_de)).keys[0];
+        const privKeyDE = (await openpgp.key.readArmored(priv_key_de)).keys[0];
+        await privKeyDE.decrypt(passphrase);
+        const encrypted = await openpgp.encrypt({
+          message: openpgp.message.fromText(plaintext),
+          publicKeys: pubKeyDE
+        });
+        privKeyDE.subKeys[0] = await privKeyDE.subKeys[0].revoke(privKeyDE.primaryKey);
+        const decOpt = {
+          message: await openpgp.message.readArmored(encrypted.data),
+          privateKeys: privKeyDE
+        };
+        await expect(openpgp.decrypt(decOpt)).to.not.be.rejected;
```

Comment:
This is a bit redundant with just calling the function. If you want to have an `expect` in there, I would write something like:
```suggestion
        const decrypted = await openpgp.decrypt(decOpt);
        expect(decrypted.data).to.equal(plaintext);
```